src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -124,21 +127,22 @@ def _DeviceAvailableCallback(closure, device, err):
 @_IssueNOCChainCallbackPythonCallbackFunct
 def _IssueNOCChainCallbackPythonCallback(devCtrl, status: PyChipError, noc: c_void_p, nocLen: int, icac: c_void_p,
                                          icacLen: int, rcac: c_void_p, rcacLen: int, ipk: c_void_p, ipkLen: int, adminSubject: int):
-    nocChain = NOCChain(None, None, None, None, 0)
+    nocChain = NOCChain(bytes(), bytes(), bytes(), bytes(), 0)
```

Comment:
I think I'd prefer if we can adjust `NOCChain` field type here (make the fields optional so they can be set to None).